### ISSUE TITLE
Multi Select cannot filter

### ISSUE THREAD

**Issue by suwubee** [twentyhq/twenty#5358]
I create MultiSelect, but cannot filter it.
![image](https://github.com/twentyhq/twenty/assets/6994794/f65fd295-a7e5-4839-ac8b-b42f80d2fee6)

**Comment by FerdiHr** [comment]
Same here, should be a nice one for a first contribution! 
It's one of the main purpose of multi-select

**Comment by FelixMalfait** [comment]
Thanks for the report!

@gitstart-twenty could you please take care of this? And do a quick audit of any other missing filters for various field type? (we don't want to be exhaustive as some don't make much business sense but we might have missed important ones like the one above ; see also https://github.com/twentyhq/twenty/issues/5216 which is ongoing)

**Comment by charlesBochet** [comment]
@FelixMalfait pg_graphql does not support filtering by Array at the moment. This ticket cannot be done until we migrate out of pg_graphql or they update their filtering feature

**Comment by FelixMalfait** [comment]
@charlesBochet seems like this was just implemented! https://github.com/supabase/pg_graphql/pull/520

**Comment by FelixMalfait** [comment]
Released yesterday: https://github.com/supabase/pg_graphql/releases/tag/v1.5.6

**Comment by martmull** [comment]
@FelixMalfait @charlesBochet @suwubee, well, think we have to wait a little bit to have enum filter support -> https://github.com/supabase/pg_graphql/issues/530

**Comment by rob-luke** [comment]
Excited to see that this issue moved to done in the product development project 🚀 . Is this issue now closed? I am very excited to use this 😄

**Comment by FelixMalfait** [comment]
Unfortunately this is still stuck due to the issue linked above by @martmull - we cannot solve this as long as the above issue isn't solved

**Comment by john-ramsey** [comment]
Potentially tangental comment but worth calling out that pg_graphql isn't supported as an extention in [GCP CloudSQL](https://cloud.google.com/sql/docs/postgres/extensions#postgresql-extensions-supported-by-cloud-sql), [Azure Database for PostgreSQL](https://learn.microsoft.com/en-us/azure/postgresql/flexible-server/concepts-extensions#extension-versions) or [AWS RDS](https://docs.aws.amazon.com/AmazonRDS/latest/PostgreSQLReleaseNotes/postgresql-extensions.html).

While likely a large item to migrate away from pg_graphql this should be added to the list of reasons.

**Comment by FelixMalfait** [comment]
@john-ramsey Yes the pg_graphql did an amazing job that allowed us to move faster initially, but long-term we'll probably have to replace because of the limitations (another one is the inability to filter by a nested object's property)

**Comment by rob-luke** [comment]
In case its useful for anyone else. I have temporarily got around this by writing a script that copies the multi select fields as text in to a text field called "multiselecthack", and then I can filter based on that. But this quickly causes you to hit this issue: https://github.com/twentyhq/twenty/issues/5056

**Comment by charlesBochet** [comment]
This will be possible as we move out of pg_graphql

**Comment by Sytten** [comment]
I would also suggest not using postgres enums as they are very limiting (you can't remove options, reorder, etc). In general it is easier to use a check constraint or verify in software (and use plain strings in db).

**Comment by charlesBochet** [comment]
@Sytten we actually already support postgres enum editing (we have some code handling it and agree it was not a piece of cake!) as data integrity is quite important for Twenty

This ticket is unblocked now :)

**Comment by ad-elias** [comment]
Could I work on this one please? :)
